app/schemas/models/thang_type.coffee

```diff
@@ -129,7 +129,7 @@ _.extend ThangTypeSchema.properties,
   scale:
     title: 'Scale'
     type: 'number'
-  spriteType: { enum: ['singular', 'segmented'], title: 'Sprite Type' }
+  spriteType: { enum: [ 'singular', 'segmented', 'animate' ], title: 'Sprite Type' }
```

Comment:
I'm not sold on the name `animate` for the new sprite type - can we think of a better one?